app/services/omdb_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class OmdbService
```

Comment:
Add specs for controller, decorator, this service.
And in general I recommend you to read about TDD and how important testing is.